examples/pendulum/pendulum.py

```diff
@@ -0,0 +1,110 @@
+import biorbd
+
+from biorbd_optim import OptimalControlProgram, OdeSolver
+from biorbd_optim.plot import PlotOcp
+from biorbd_optim.problem_type import ProblemType
+from biorbd_optim.objective_functions import ObjectiveFunction
+from biorbd_optim.constraints import Constraint
+from biorbd_optim.path_conditions import Bounds, QAndQDotBounds, InitialConditions
+
+
+def prepare_ocp(biorbd_model_path="eocar.bioMod"):
+    # --- Options --- #
+    # Model path
+    biorbd_model = biorbd.Model(biorbd_model_path)
+
+    # Problem parameters
+    number_shooting_points = 30
+    final_time = 2
+    torque_min, torque_max, torque_init = -100, 100, 0
+
+    # Add objective functions
+    objective_functions = ((ObjectiveFunction.minimize_torque, 100),)
+
+
+    # Dynamics
+    variable_type = ProblemType.torque_driven
+
+    # Constraints
+    constraints = ()
+
+    # Path constraint
+    X_bounds = QAndQDotBounds(biorbd_model)
+
+    for i in range(biorbd_model.nbQ() + biorbd_model.nbQdot()):
+        X_bounds.first_node_min[i] = 0
+        X_bounds.last_node_min[i] = 0
+        X_bounds.first_node_max[i] = 0
+        X_bounds.last_node_max[i] = 0
```

Comment:
It feels that this for loop is useless because you overwrite all the data afterwards... 
I think it should be removed. Also, could you regroup by time instead of min/max, it is confusing to me that first_node_min is not with first_node_max!

Reply:
What if I only change the Qdot_translation at the final node and keep the loop?

    for i in range(biorbd_model.nbQ() + biorbd_model.nbQdot()):
        X_bounds.first_node_min[i] = 0
        X_bounds.first_node_max[i] = 0
        X_bounds.last_node_min[i] = 0
        X_bounds.last_node_max[i] = 0


    #Q_rotation
    X_bounds.last_node_min[1] = 3.14
    X_bounds.last_node_max[1] = 3.14